components/Button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import StyledButton from './Button.styles';
+
+const Button = ({children, url, htmlAttributes }) => (
+  <StyledButton src={url} attributes={htmlAttributes} >{children}</StyledButton>
```

Comment:
I'm not behind passing `{children}` here

```
<Button htmlAttributes={{ width: 100 }}>
  <div>I'm a div that shouldn't be allowed in here</div>
</Button>
```

Is there a reason behind this decision? Bit nervous of things like above happening